backtracking/minimax.py

```diff
@@ -35,36 +50,45 @@ def minimax(
     12
     """
 
+    # Check for invalid inputs
```

Comment:
```suggestion
```
I feel that this comment isn't really necessary. The if-statement is very simple so it's already pretty clear what it does.